### ISSUE TITLE
Timeseries example for RadChart?

### ISSUE THREAD

**Issue by brylie** [telerik/nativescript-ui-feedback#997]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
_Yes/No_

### Tell us about the problem
_Please, ensure your title is less than 63 characters long and starts with a capital
letter._

### Which platform(s) does your issue occur on?
_iOS/Android/Both (if applicable tell us the specific version of the platform)_

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** (the version of nativescript-telerik-ui/-pro in the `package.json`)
- **CLI:** (run `tns --version` to fetch it)
- **Cross-platform modules:** (the version of tns-core-modules in the `package.json`)
- **Runtime(s):** (the `"tns-android"` and `"tns-ios"` properties in the `package.json`)

### Please tell us how to recreate the issue in as much detail as possible.
1. Start the application ..
2. ...

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
_(You can paste entire code snippets or attach a runnable project)_

**Comment by brylie** [comment]
I am using Plotly as a benchmark here, after coming up significantly short with several other charting libraries. 

https://plot.ly/javascript/

Might it be possible to use Plotly as the recommended charting library, rather than going down the rabbit hole, which has considerable nuance, of creating yet another charting library?

**Comment by tgpetrov** [comment]
Hi @brylie 
Thank you for the feedback.
Is [this](https://github.com/telerik/nativescript-ui-samples/blob/master/chart/app/examples/axes/date-time-axes/date-time-axes-page.xml) the example with the LineSeries presenting continuous data that you are looking for? Here's how it looks:
<img src="https://user-images.githubusercontent.com/6085251/53227671-9a605480-3687-11e9-834b-348a45e24d1a.png" width="300px"/>

**Comment by brylie** [comment]
Yes, basically showing a date axis with a data axis (numeric). In general, each chart type is suitable for differing types of data, numeric, categorical, etc., so using the example data should be appropriate for the chart type. I also realize there is quite a lot of underlying complexity in date handling/rendering for timeseries charts, which is why it might be good to use an existing charting library that has already paved the way.

**Comment by tgpetrov** [comment]
Great, so if I understand correctly, we have an example of a LineChart using combination of Date and Number axes, but we need to improve docs and examples so that such examples are more easily discoverable. Also we should prefer more meaningful combinations of axes and series (i.e. connecting values in discrete categories with LineSeries is not the most popular scenario for these series)?

**Comment by brylie** [comment]
Yep, but I can't find the example you mention in the [line chart documentation](https://docs.nativescript.org/ui/professional-ui-components/Chart/Series/line). Rather, the line chart documentation reuses the same sample data that is more appropriate for a [bar chart example](https://docs.nativescript.org/ui/professional-ui-components/Chart/Series/bar). I.e. there isn't a meaningful way to connect 'Germany', 'France', 'Bulgaria', etc. as they are distinct categories.

See: [Data: Continuous vs. Categorical](https://eagereyes.org/basics/data-continuous-vs-categorical)

**Comment by tgpetrov** [comment]
You are right. Thanks again for the feedback.

**Comment by djmgh** [comment]
I am also very confused by the choice of examples for LineSeries and SplineSeries, and generally how to translate the chart examples to Vue. An example plotting an unordered list of x,y numbers would be appreciated

**Comment by brylie** [comment]
@tgpetrov would it be possible to integrate a more mature, and focused, charting library into NativeScript UI via JavaScript? For example, [Plotly.js](https://plot.ly/javascript/), [ECharts](https://ecomfe.github.io/echarts-doc/public/en/index.html), and [TauCharts](https://taucharts.com/) are all fairly mature with good APIs. 

It sincerely seems that the NativeScript chart library is somewhat an afterthought, and it would be beneficial to defer to a project with holistic design and clear focus for data visualization.

**Comment by VladimirAmiorkov** [comment]
Hi @brylie ,

Simply integrating a JS library into the native rendering of the _nativescript-ui-chart_ is a relatively complex task. The current implementation is entirely custom both the logic and the rendering is done in native Objective-C code for iOS and Java for Android which means that there is no easy way to integrate a third party JS library which has been designed to use the DOM for rendering its elements. For now we are focused on resolving any ongoing issues with the  _nativescript-ui-chart_ and we always welcome contributions via our DNA process that can be found [here](https://github.com/NativeScript/nativescript-ui-feedback#interested-in-contributing-to-the-codebase).
